translations/de/pack/eoe/eoep.json

```diff
@@ -511,7 +511,7 @@
     {
         "code": "08069",
         "name": "In sich ruhend",
-        "text": "Schnell. Nur 1 [[Gelassenheit]] im Spiel.\nDu bekommst +1 auf deinen Fertigkeitswert während Fertigkeitsproben auf [[Zauber]]-Karten.\nNicht-direkter Schaden/Horror muss In sich ruhend zugewiesen werden, bevor er deiner Ermittlerkarte zugewiesen werden kann.\n[fast] Gib während einer Fertigkeitsprobe auf einer [[Zauber]]-Karte, gib 1 Ressource aus: Du bekommst +1 auf deinen Fertigkeitswert für diese Probe.",
+        "text": "Schnell. Nur 1 [[Gelassenheit]] im Spiel.\nDu bekommst +1 auf deinen Fertigkeitswert während Fertigkeitsproben auf [[Zauber]]-Karten.\nNicht-direkter Schaden/Horror muss In sich ruhend zugewiesen werden, bevor er deiner Ermittlerkarte zugewiesen werden kann.\n[fast] Gib während einer Fertigkeitsprobe auf einer [[Zauber]]-Karte aus: Du bekommst +1 auf deinen Fertigkeitswert für diese Probe.",
```

Comment:
There is something missing now:
"... Gib während einer Fertigkeitsprobe auf einer [[Zauber]]-Karte **1 Ressource** aus ..."

Reply:
Woops, you're right! Thanks, I'll fix that.